jenkins.sh

```diff
@@ -109,6 +109,7 @@ builder_run 'Build' 0 1 ./build.sh
 builder_run 'Tests' 0 1 ./test.sh
 builder_run 'GUI-Tests' 1 1 ./test-gui.sh crud
 builder_run 'GUI-Tests-S2I' 1 0 ./test-gui.sh s2i
+builder_run 'GUI-Tests-DeployImage' 1 0 ./test-gui.sh deployImage
```

Comment:
I think we should probably change GUI-Tests-S2I to GUI-Tests-New-App (or similar), then include these tests in that suite.